data/json/construction.json

```diff
@@ -2371,5 +2371,23 @@
         ],
         "pre_terrain" : "t_reinforced_glass",
         "post_terrain" : "t_reinforced_glass_shutter"
+    },{
+        "type" : "construction",
+        "description" : "Build makeshift nest",
+        "category" : "FURN",
+        "required_skills" : [ [ "fabrication", 1 ] ],
+        "time" : 35,
+        "qualities": [
+            [
+            { "id": "HAMMER", "level": 2 }
+            ]
+            ],
```

Comment:
The linting tool doesn't like the fabrication field, so this time I'm going to pass it through without that field, and then add it later.